shared/css/fonts/noto-sans.scss

```diff
@@ -0,0 +1,75 @@
+/*
+ * File is used in dashboard/app/assets/stylesheets/application.scss
+ * See: https://fonts.google.com/noto/fonts?noto.query=Noto+Sans
+ *
+ * 100 – Thin
+ * 200 – Extra Light (Ultra Light)
+ * 300 – Light
+ * 400 – Regular (Normal)
+ * 500 – Medium
+ * 600 – Semi Bold (Demi Bold)
+ * 700 – Bold
+ * 800 – Extra Bold (Ultra Bold)
+ * 900 – Black (Heavy)
+ */
+
+ // Latin, Cyrillic and Greek:
+ // az-AZ, bs-BA, ca-ES, co-FR, cs-CZ, da-DK, de-DE, el-GR, en-GB, en-US, es-ES, es-MX, et-EE, eu-ES, fi-FI, fil-PH,
+ // fr-FR, ga-IE, gl-ES, haw-HI, hr-HR, hu-HU, id-ID, is-IS, it-IT, kk-KZ, ky-KG, lt-LT, lv-LV, mi-NZ, mk-MK, ms-MY,
+ // mt-MT, nl-NL, nn-NO, no-NO, pl-PL, pt-BR, pt-PT, ro-RO, ru-RU, se-FI, sk-SK, sl-SI, sm-WS, sq-AL, sr-SP, sv-SE,
+ // tg-TJ, tr-TR, uk-UA, uz-UZ, vi-VN, zu-ZA
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans:ital,wdth,wght@0,62.5..100,100..900;1,62.5..100,100..900&display=swap');
```

Comment:
Would these imports create a network request to Google to fetch the fonts or would these be served by Code.org? If it does go to Google we should monitor Zendesk and eyes for font download failures

Reply:
Looking at metropolis, we base64 encode the font so we self-deliver whereas in this one it pulls from the Google Font CDN. This is probably fine to start with, but we may need to look into self-serving them due to school network restrictions

Reply:
The font import is served by Google Fonts servers, not Code.org. Therefore, it will not impact Code.org's network load.

Reply:
The font import is served by Google Fonts servers, not Code.org. Therefore, it will not impact Code.org's network load.

Reply:
cc @code-dot-org/infrastructure

Reply:
My comment was mostly regarding how schools sometimes block certain domains which would result in the font being blocked. However, I think the net result would be better (a blocked font would result in the same behavior we have today) so I think this is good and we'll see how it plays out.

Reply:
Have we looked into whether we could self-host these new fonts?

Reply:
@code-dot-org/infrastructure Should we self-host these fallback fonts, or is it okay to source them from Google?
As Stephen mentioned, the worst-case scenario is that a blocked font would result in the same behavior we experience today.

Reply:
@code-dot-org/infrastructure Should we self-host these fallback fonts, or is it okay to source them from Google? As Stephen mentioned, the worst-case scenario is that a blocked font would result in the same behavior we experience today.

Reply:
cc @Hamms  @snickell

Reply:
I think we can self-host them but we can do self-hosting as a separate fast follow Jira task (no need to block on this task IMO). We would download the noto fonts and store them at the same place we do for Metropolis.